### PR TITLE
Add resource file for sway

### PR DESCRIPTION
**Pull request by ranisalt**
Adds support for the [sway](https://github.com/swaywm/sway) window manager for Wayland. Although similar to i3, since the i3 config gets loaded from Xresources file, it does not work with Wayland. Conversely, I used tne include feature of sway that does not exist in i3 to load a file with all colors and background set.

Usage:
- In your `.config/sway/config` file, add before any color usage:
```
include "$HOME/.cache/wal/colors-sway"
```
It sets `$wallpaper` to the wallpaper path, `$background`, `$foreground` and `$color0` through `$color15`.

- To use the wallpaper:
```
output <name> background $wallpaper <mode>`
```

- To set colors:
```
client.focused $color0 $bg $fg $color7
```

It reloads the colorscheme when loading/reloading colors, too.

### CODE REVIEW

File: pywal/__main__.py [hunk 1]
```diff
@@ -69,7 +69,7 @@ def get_args(args):
                      help="Print \"wal\" version.")
 
     arg.add_argument("-e", action="store_true",
-                     help="Skip Reloading Environment gtk/xrdb/i3/polybar")
+                     help="Skip Reloading Environment gtk/xrdb/i3/sway/polybar")
```
Comment (dylanaraps) [inline pywal/__main__.py]: This line is one character too long. Rename `Environment` to `Programs`.

**Comment by dylanaraps** [comment]
LGTM :+1:

Could you also add the usage info to the wiki?

File: pywal/__main__.py (hunk 1 above)
Comment (ranisalt) [reply pywal/__main__.py]: Does it even need to have a word? I think "Skip reloading gtk/xrdb/i3/sway/polybar" is clear enough.

**Comment by ranisalt** [comment]
@dylanaraps done both, sorry for the delay

**Comment by dylanaraps** [comment]
Merged, thanks!

**Pull request merged**